packages/recommendations-vdom/src/ListView.tsx

```diff
@@ -0,0 +1,40 @@
+/** @jsx createElement */
+
+import {
+  RecommendationClassNames,
+  RecommendationTranslations,
+  RecordWithObjectID,
+  Renderer,
+  ViewProps,
+} from './types';
+import { cx } from './utils';
+
+export function createListViewComponent({ createElement }: Renderer) {
+  return function ListView<TItem extends RecordWithObjectID>(
+    props: ViewProps<
+      TItem,
+      RecommendationTranslations,
+      RecommendationClassNames
+    >
+  ) {
+    return (
+      <div
+        className={cx(
+          'auc-Recommendations-container',
+          props.classNames.container
+        )}
+      >
+        <ol className={cx('auc-Recommendations-list', props.classNames.list)}>
+          {props.items.map((item) => (
+            <li
+              key={item.objectID}
+              className={cx('auc-Recommendations-item', props.classNames.item)}
+            >
+              <props.itemComponent item={item} />
```

Comment:
theres' another prop called props.Header (capitalised), maybe it's nicer if all components that are props are pascal case? IIRC that's a requirement if it's not read from an object directly, so being consistent can help

Reply:
I agree we need to find some consistency. We also have `fallbackComponent`. I think it's out of scope for this PR but let's definitely address this before public release.

Reply:
I'll let you open an issue / note it down somewhere and this can be resolved

Reply:
[Created a ticket on JIRA](https://algolia.atlassian.net/browse/DX-2016).